dapp/src/twitter.ts

```diff
@@ -24,6 +44,8 @@ export const verifyTweet = async (userData, twitterHandle, tweetURL) => {
         variant: 'success',
       });
 
+      let temp = await res.text();
```

Comment:
?

Reply:
I don't understand what this does, the next line is identical?

Reply:
Removed